_custom_build/backend.py

```diff
@@ -1,6 +1,6 @@
 import sys
 
-from setuptools.build_meta import *  # noqa: F401, F403
```

Comment:
Why does Flake8 warn about F401 when Ruff doesn't?

Reply:
I suppose Flake8 warning about both F401 and F403 for a star import is overkill anyway.